### ISSUE TITLE
Create a Google Maps map block for [bw_show_googlemap]

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-blocks#3]
### Requirement
Create a Google Maps map block to replace/enhance [bw_show_googlemap]

### Initial solution
- Dummy content in the editor
- Displays the GoogleMap using the [bw_show_googlemap] shortcode in the front end.

**Comment by bobbingwide** [comment]
Here's a screenshot of a 'confused' Google Maps block.

It's confused because it doesn't have a border.  

It was created using the trick I learnt that automatically corrects a block when it's selected as part of a group. I discovered this trick when selecting and moving a heading, paragraph and block list block from the bottom of a page to the top. 
I tried the same technique with a single broken block ( this Google Maps block ) and it went dolally.

![image](https://user-images.githubusercontent.com/2474435/72373208-b9ccf980-36ff-11ea-9c9f-6776869f6b42.png)
